src/Qir/Tests/QIR-static/qir-test-math.cpp

```diff
@@ -98,6 +99,11 @@ TEST_CASE("QIR: Math.IeeeRemainder", "[qir.math][qir.Math.IeeeRemainder]")
     REQUIRE(0 == Microsoft__Quantum__Testing__QIR__Math__IeeeRemainderTest__Interop());
 }
 
+TEST_CASE("QIR: Math.Exponent.builtin", "[qir.math][qir.Math.Exponent.builtin]")
+{
+    REQUIRE(0 == Microsoft__Quantum__Testing__QIR__Math__ExponentTest__Interop());
```

Comment:
Looks like Microsoft__Quantum__Testing__QIR__Math__ExponentTest__Interop() and ExponentTest() should be related, but I can't understand how...